friends/src/components/FriendList.css

```diff
@@ -0,0 +1,39 @@
+.friend {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 15px;
+}
+
+ul {
+  list-style-type: none;
+}
+
+li {
+  margin-right: 10px;
+}
+
+button {
+  width: 70px;
+}
+
+div form {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+label {
+  margin-right: 10px;
+}
+
+span {
+  font-weight: bold;
+}
+
+form span {
+  margin-right: 10px;
```

Comment:
how many `margin-right: 10px;`s do you need? Be ever watchful for patterns 👁 👁‍🗨